exercises/complex-numbers/canonical-data.json

```diff
@@ -0,0 +1,88 @@
+{
+  "exercise": "complex-numbers",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "comments": [
+        " The canonical data assumes mathematically correct real   ",
+        " numbers. The testsuites should consider rounding errors  ",
+        " instead of testing for exact values for any non-integer  ",
+        " tests.                                                   "
+      ],
+      "description": "Arithmetic",
+      "cases": [
+        {
+          "description": "Addition",
+          "property": "add",
+          "z1": "1+2i",
+          "z2": "3+4i",
+          "expected": "4+6i"
+        },
+        {
+          "description": "Subtraction",
+          "property": "sub",
+          "z1": "1+2i",
+          "z2": "3+4i",
+          "expected": "-2-2i"
+        },
+        {
+          "description": "Multiplication",
+          "property": "mul",
+          "z1": "1+2i",
+          "z2": "3+4i",
+          "expected": "-5+10i"
+        },
+        {
+          "description": "Division",
+          "property": "div",
+          "z1": "1+2i",
+          "z2": "3+4i",
+          "expected": "11/25+2i/25"
+        }
+      ]
+    },
+    {
+      "description": "Absolute value",
+      "property": "abs",
+      "input": "(1+i)*(1+i)",
```

Comment:
Good point, I chose that because it gave a round result that would be easy to check by hand. Changing it to `abs(3+4i)=5`.